tutorials/huggingface_24sparse_example.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
Really not a fan of checking in notebooks, mind turning this into a python file, there should be a button to do this in jupyter, alternatively link to the tutorial hosted on google colab

Reply:
Sure, that fine with me.